MAINTAINERS.md

```diff
@@ -0,0 +1,11 @@
+Maintainers List
+================
+
+For notes on how to read this information, please refer to `MAINTAINERS.md` in
+the main Unikraft repository.
+
+	LIBSODIUM-UNIKRAFT
+	M:	Felipe Huici <felipe.huici@neclab.eu>
+	M:	Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>
+	L:	minios-devel@lists.xen.org
+	F: *
```

Comment:
This file can be deleted, we are about to deprecate this file from the Unikraft ecosystem :) We are working on a new governance model (and repo) which will superced this.

Reply:
Done